logstash-core/lib/logstash/config/mixin.rb

```diff
@@ -142,6 +142,11 @@ def config_init(params)
       instance_variable_set("@#{key}", value)
     end
 
+    # now that we know the parameters are valid, we can obfuscate the original copy
```

Comment:
what about changing `obfuscate` to `hide` when talking about security, obfuscate has this idea of  security through obscurity, what do you think?